lld/test/ELF/aarch64-call26-thunk-execute-only.s

```diff
@@ -0,0 +1,21 @@
+// REQUIRES: aarch64
+// RUN: llvm-mc -filetype=obj -triple=aarch64 %s -o %t.o
+// RUN: ld.lld %t.o --defsym big=0x1111222233334444 -o %t
+// RUN: llvm-objdump -d --no-show-raw-insn %t | FileCheck %s
+
+.section .text,"axy",@progbits,unique,0
+.globl _start
+_start:
+    bl big
+
+// CHECK: Disassembly of section .text:
+// CHECK-EMPTY:
+// CHECK-LABEL: <_start>:
+// CHECK-NEXT:     210120: bl      0x210124
```

Comment:
omit the insignificant address and `bl {{.*}} <__AArch64AbsXOLongThunk_big>` so that the test will not need an update when address layout changes